settings/staging.py

```diff
@@ -0,0 +1,17 @@
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+
+from pennlabs.settings.production import *
+
+
+DEBUG = True
+
+PLATFORM_ACCOUNTS.update(
+    {
+        'REDIRECT_URI': os.environ.get('LABS_REDIRECT_URI', f'https://{BACKEND_DOMAIN}/accounts/callback/'),
```

Comment:
Does the formatted string open us up to RCE? Since they can run straight python code?